<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDialog</class>
 <widget class="QDialog" name="HelpDialog">
  <property name="windowTitle">
   <string>–°–ø—Ä–∞–≤–∫–∞ OCRtoODT</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">

   <!-- –í–µ—Ä—Ö–Ω—è—è —á–∞—Å—Ç—å: –ª–æ–≥–æ—Ç–∏–ø + –∑–∞–≥–æ–ª–æ–≤–æ–∫ -->
   <item>
    <layout class="QHBoxLayout" name="layoutHeader">
     <item>
      <widget class="QLabel" name="lblLogo">
       <property name="pixmap">
        <pixmap>resources/icons/help.svg</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="text">
        <string>&lt;h2&gt;OCRtoODT ‚Äî –¥–æ–∫—É–º–µ–Ω—Ç–∞—Ü–∏—è&lt;/h2&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- –ì–æ—Ä–∏–∑–æ–Ω—Ç–∞–ª—å–Ω–∞—è –ª–∏–Ω–∏—è -->
   <item>
    <widget class="Line" name="lineHeader">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>

   <!-- –û—Å–Ω–æ–≤–Ω–∞—è –æ–±–ª–∞—Å—Ç—å —Å–æ —Å–ø—Ä–∞–≤–∫–æ–π -->
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>

   <!-- –ü–æ–∏—Å–∫–æ–≤–∞—è –ø–∞–Ω–µ–ª—å -->
   <item>
    <layout class="QHBoxLayout" name="layoutSearch">
     <item>
      <widget class="QLabel" name="lblFind">
       <property name="text">
        <string>–ù–∞–π—Ç–∏:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFind"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrev">
       <property name="text">
        <string>‚Üê</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>‚Üí</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- –ü–æ–¥–ø–∏—Å—å Ctrl+F -->
   <item>
    <widget class="QLabel" name="lblHint">
     <property name="text">
      <string>üí° –°–æ–≤–µ—Ç: –∏—Å–ø–æ–ª—å–∑—É–π—Ç–µ Ctrl+F –¥–ª—è —Ñ–æ–∫—É—Å–∞ –Ω–∞ –ø–æ–∏—Å–∫</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #6b7280; font-size: 9pt;</string>
     </property>
    </widget>
   </item>

   <!-- –ö–Ω–æ–ø–∫–∞ –ó–∞–∫—Ä—ã—Ç—å -->
   <item alignment="Qt::AlignRight">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
